include/fiction/technology/real_position.hpp

```diff
@@ -0,0 +1,25 @@
+//
+// Created by Jan Drewniok on 12.12.22.
+//
+
+#ifndef FICTION_REAL_POSITION_HPP
+#define FICTION_REAL_POSITION_HPP
+
+
+#include "fiction/algorithms/simulation_sidb/simulation_parameter.hpp"
+#include "fiction/traits.hpp"
+
+namespace fiction
+{
+
+template <typename Lyt, typename Dist = double>
+std::pair<Dist, Dist> real_position(const cell<Lyt> &c)
```

Comment:
Same here: unclear what this function does by its name. Please rename with something more verbose